assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,215 @@
+	},
+};
+
+export const Error = WidgetAreaTemplate.bind();
+Error.storyName = 'Error';
+Error.args = {
+	setupRegistry: ( { dispatch } ) => {
+		const error = {
+			code: 'missing_required_param',
+			message: 'Request parameter is empty: metrics.',
+			data: {},
+		};
+		dispatch( STORE_NAME ).receiveError( error, 'getReport', [ options ] );
+		dispatch( STORE_NAME ).finishResolution( 'getReport', [ options ] );
+	},
+};
+
+export const LoadedEntityURL = WidgetAreaTemplate.bind();
+LoadedEntityURL.storyName = 'Loaded with entity URL set';
+LoadedEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+	},
+};
+
+export const LoadingEntityURL = WidgetAreaTemplate.bind();
+LoadingEntityURL.storyName = 'Loading with entity URL set';
+LoadingEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsCompareEntityURL ] );
+	},
+};
+
+export const DataUnavailableEntityURL = WidgetAreaTemplate.bind();
+DataUnavailableEntityURL.storyName = 'Data Unavailable with entity URL set';
+DataUnavailableEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( [], { options: optionsEntityURL } );
+	},
+};
+
+export const ErrorEntityURL = WidgetAreaTemplate.bind();
+ErrorEntityURL.storyName = 'Error with entity URL set';
+ErrorEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+		const error = {
+			code: 'missing_required_param',
+			message: 'Request parameter is empty: metrics.',
+			data: {},
+		};
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveError( error, 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).finishResolution( 'getReport', [ optionsEntityURL ] );
+	},
+};
+
+export default {
+	title: 'Modules/Analytics/Widgets/DashboardUniqueVisitorsWidget',
+	decorators: [
+		( Story ) => {
+			const setupRegistry = ( registry ) => {
+				const { dispatch } = registry;
+
+				const [ property ] = accountsPropertiesProfiles.properties;
+				registry.dispatch( STORE_NAME ).receiveGetSettings( {
```

Comment:
Here `dispatch` is destructured, but `registry.dispatch` is also called. Since `registry` is used below, we should just use `registry.` methods on all as mentioned above.

Reply:
👍